aws/resource_aws_ecs_service.go

```diff
@@ -69,6 +69,13 @@ func resourceAwsEcsService() *schema.Resource {
 				Default:  "EC2",
 			},
 
+			"platform_version": {
+				Type:     schema.TypeString,
+				ForceNew: false,
+				Optional: true,
+				Default:  "LATEST",
```

Comment:
Defaulting this to `LATEST` breaks all the other tests with `launch_type` set to `EC2` (or unset, which defaults to `EC2`) with the following:

```
--- FAIL: TestAccAWSEcsService_withARN (4.53s)
    testing.go:538: Step 0 error: Error applying: 1 error occurred:
        	* aws_ecs_service.mongo: 1 error occurred:
        	* aws_ecs_service.mongo: InvalidParameterException: The platform version must be null when specifying an EC2 launch type.
```

My recommendation would be to adjust this attribute's schema to the following:

```go
"platform_version": {
	Type:     schema.TypeString,
	Optional: true,
	Computed: true,
},
```

The `Computed: true` here will address your original concern by allowing Terraform plans to ignore any difference if the argument is not defined in the Terraform configuration.